programs/cross-language-coverage/README.md

```diff
@@ -0,0 +1,74 @@
+# Cross-language Implementation Quality
+
+This directory collects metrics to assist in confirming the presence of (at least one) "high quality" implementations across programming languages.
+
+Absence from this directory is not meant to be a slight against anyone or anyone's work certainly.
+
+The goals are to identify gaps in the ecosystem that might need assistance, and ultimately to help the JSON Schema ecosystem and its users in finding solid, compliant implementations.
+
+The intial collection of TODO programming languages which appears below is simply a starting point, based largely on:
+
+  * the [GitHub popularity list](https://octoverse.github.com/2022/top-programming-languages)
+  * the [TIOBE index](https://www.tiobe.com/tiobe-index/)
+
+where it combines languages at the top of the popularity list with those on the rising / fastest growing list.
+
+You can help this exercise!
+For each language, we wish to track whether one or more known implementations exist which:
+
+  * Fail less than 2% (around 30) tests from the official JSON Schema test suite
+    (This 2% number was chosen simply because it represents the dividing point between one cluster of implementations supported by Bowtie.
+     Above this number is a collection of implementations failing larger orders of magnitude of tests).
+  * Supports one of the most recent 2 releases of JSON Schema
+  * Has had a release of itself in the last 3 years (which is around the timeline on which JSON Schema itself has seen its last release)
```

Comment:
I think if we say "has had a release since the last spec release" that goes very quickly "badly" right after a JSON Schema release where nearly no implementations will have a release right after.
I picked 3 mostly "arbitrarily" given I think *we* are an active project but haven't released in 3 years, so I didn't want to expect more from implementations, but otherwise intended to *stick* with 3.

Open to suggestions of course though, this is just a fuzzy attempt. Maybe even deleting the end of the sentence (the parenthetical) is reasonable?

Reply:
I think the "3 years" is fine.  Just noticing that the "about the same as the last release" will go stale.  If your target is the 3 years, then I think just having that and not relating it to the spec is fine.

Reply:
(Makes sense, removing.)